vmdb/spec/models/ems_refresh/refreshers/openstack_refresher_rhos_grizzly_spec.rb

```diff
@@ -67,7 +67,7 @@ def assert_table_counts
     SystemService.count.should       == 0
 
     Relationship.count.should        == 13
-    MiqQueue.count.should            == 19
+    MiqQueue.count.should            == 20
```

Comment:
@blomquisg is this what we want to test for?  the fact that @Ladas had to change this seems to indicate these row count assertions are brittle.  I'd prefer to enumerate which of the 19/20 queue messages we really care about and do a .count on those specific conditions to verify we get the number of rows we care about.